scrabble.js

```diff
@@ -1,16 +1,228 @@
+// Start Scores Object
+const scores = {
+  'A': 1,
+  'E': 1,
+  'I': 1,
+  'O': 1,
+  'U': 1,
+  'L': 1,
+  'N': 1,
+  'R': 1,
+  'S': 1,
+  'T': 1,
+  'D': 2,
+  'G': 2,
+  'B': 3,
+  'C': 3,
+  'M': 3,
+  'P': 3,
+  'F': 4,
+  'H': 4,
+  'V': 4,
+  'W': 4,
+  'Y': 4,
+  'K': 5,
+  'J': 8,
+  'X': 8,
+  'Q': 10,
+  'Z': 10
+}
 
+const letters = [
+  'A', 'A', 'A', 'A', 'A', 'A', 'A', 'A', 'A',
+  'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E',
+  'I', 'I', 'I', 'I', 'I', 'I', 'I', 'I', 'I',
```

Comment:
While this certainly works, this data structure isn't super readable. At a glance, how many `I`s are there?

Instead, you might use a hash to map letter -> tile count, and then write a function to generate this array.